website/blog/2023-10-12-build-internal-apps-illa-cloud-ferretdb.md

```diff
@@ -0,0 +1,141 @@
+---
+slug: build-internal-apps-illa-cloud-ferretdb
+title: How to Build Internal Apps with Illa Cloud and FerretDB
+authors: [alex]
+description: >
+  In this blog post, we will explore ways to quickly build internal apps in minutes using Illa Cloud and FerretDB.
+image: /img/blog/ferretdb-illacloud.jpg
+tags: [compatible applications, tutorial, open source]
+---
+
+![How to Build Internal Apps Dashboards with Illa Cloud and FerretDB](/img/blog/ferretdb-illacloud.jpg)
+
+In this blog post, we will explore ways to quickly build internal apps in minutes using [Illa Cloud](https://www.illacloud.com/) and [FerretDB](https://www.ferretdb.io/).
+
+<!--truncate-->
+
+FerretDB is an [open-source document database replacement](https://blog.ferretdb.io/open-source-is-in-danger/) for MongoDB that converts MongoDB protocols to SQL and provides [PostgreSQL](https://www.postgresql.org/) and [SQLite](https://www.sqlite.org/) as database backend options.
```

Comment:
"for MongoDB that converts MongoDB protocols to SQL " 
I see that we started using this sentence very often, but does it make sense?
Wouldn't it be better to say "at FerretDB, we turn Postgres or SQLite into a MongoDB compatible database, using these relational databases as the backend"

Reply:
"converts MongoDB protocols to SQL" is also technically incorrect – it is not possible to convert protocols (and there is only one, not many) into a query language